src/lib.rs

```diff
@@ -97,6 +97,16 @@
 //!
 //! Check out the [custom_key](https://github.com/AaronErhardt/actix-governor/blob/main/examples/custom_key.rs) example to see how a custom key extractor can be implemented.
 //!
+//! # Customize response error content
+//!
+//! By default, when the rate limit is exceeded the error will show up is `Too many requests, retry in {}s`<br>
+//! and content type it's plaintext, if you want to customize it you need to override the [`response_error_content`] function<br>
```

Comment:
The same thing here. I don't thing the `<br>` makes sense here anyway.

Reply:
I put it to write the line below it, if I didn't put it he would stifle it with the same line, and if I left a blank line the space would be as big as the distance between them and the example

> 
![image](https://user-images.githubusercontent.com/59842932/184238679-10732f21-e7e1-4984-a21e-bb09f8df52f7.png)

Reply:
But that's the idiomatic way of writing Rust documentation / MarkDown. A paragraph should let the text flow naturally. Manual `<br>` tags only make things look odd on devices with small screens. On top of that, placing a manual `<br>` in the middle of a sentence doesn't make any sense either.